sync2jira/main.py

```diff
@@ -152,7 +138,7 @@ def load_config(loader=fedmsg.config.load_config):
 
 def listen(config):
     """
-    Listens to activity on upstream repos on pagure and github \
+    Listens to activity on upstream repos on github \
     via fedmsg, and syncs new issues there to the JIRA instance \
     defined in 'fedmsg.d/sync2jira.py'
```

Comment:
GitHub should, generally speaking, be spelled with two capital letters (at least in English text).

Also, we shouldn't need the line continuation (backslash) characters in a docstring.

Reply:
I'll take care of this

Reply:
see [40a3c50](https://github.com/release-engineering/Sync2Jira/pull/208/commits/40a3c505c426a057771a28ec526f832a0e3d663f)